### PR TITLE
Update deployment for celo-blockchain 1.5.x

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description

Update testnet package with new configuration included in 1.5.x:
- Updated `pprof` options to new format
- Updated `rpc` options to new format (`http`)
- Updated `ws` options to new format
- Added `http.timeout` options for private nodes.

### Tested

Deployed in baklava and alfajores successfully. 

### Backwards compatibility

Helm charts maintain compatibility with geth versions <=1.5. For terraform modules, this change require using geth >=1.5 (terraform templates are not actively used by any clabs existing infrastucture at the moment).

### CODE REVIEW

File: packages/helm-charts/common/templates/_helpers.tpl [hunk 1]
```diff
@@ -535,3 +532,21 @@ prometheus.io/port: "{{ $pprof.port | default 6060 }}"
...
+{{- if (or .Values.metrics .Values.pprof.enabled) | default false }}
+# Check the format of pprof cmd arguments
+set +e
+geth --help | grep 'pprof.port' >/dev/null
+pprof_new_format=$?
+set -e
+if [ $pprof_new_format -eq 0 ]; then
+  ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --pprof --pprof.port {{ .Values.pprof.port | default "6060" }} --pprof.addr 0.0.0.0"
+else
+  ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --pprof --pprofport {{ .Values.pprof.port | default "6060" }} --pprofaddr 0.0.0.0"
```
Comment (trianglesphere) [inline packages/helm-charts/common/templates/_helpers.tpl:547]: I feel like we may have supported both options for a while and may not need to have this if statement. Let me double check which version was the first to support this.

**Comment by trianglesphere** [comment]
This is working with 1.5.0-beta.1 with `celotool deploy upgrade testnet -e baklava`

File: packages/helm-charts/common/templates/_helpers.tpl (hunk 1 above)
Comment (trianglesphere) [reply packages/helm-charts/common/templates/_helpers.tpl:547]: I looked, and this is supported after v1.4. We probably have some code on 1.3, but after espresso, we should be able to remove this.

**Review by trianglesphere — APPROVED** [review]

File: packages/helm-charts/common/templates/_helpers.tpl (hunk 1 above)
Comment (jcortejoso) [reply packages/helm-charts/common/templates/_helpers.tpl:547]: The rpc/ws/http flagas has been also updated. We can keep this for now, and drop compatibility with <1.5 after the hardfork activation.

**Review by eruizgar91 — APPROVED** [review]

File: packages/helm-charts/load-test/templates/load-test.statefulset.yaml [hunk 2]
```diff
@@ -154,12 +151,10 @@ spec:
 
           ACCOUNT_ADDRESS=$(awk 'NR=={{ add $index 1 }}' /root/.celo/address)
 
+{{ include  "common.geth-http-ws-flags" (dict "Values" $.Values "rpc_apis" "eth,web3,debug,admin,personal,net" "ws_port" "8545" "listen_address" "0.0.0.0") | indent 10 }}
+
           exec geth \
           --nousb \
-          --rpc \
```
Comment (gastonponti) [inline packages/helm-charts/load-test/templates/load-test.statefulset.yaml:159]: Shouldn't we add the `--http` flag here?

File: packages/helm-charts/load-test/templates/load-test.statefulset.yaml [hunk 3]
```diff
@@ -109,14 +109,12 @@ spec:
           ACCOUNT_ADDRESSES=$(cat /root/.celo/address | tr '\n' ',')
           ACCOUNT_ADDRESSES=${ACCOUNT_ADDRESSES::-1}
 
+{{ include  "common.geth-http-ws-flags" (dict "Values" $.Values "rpc_apis" "eth,web3,debug,admin,personal,net" "ws_port" "8545" "listen_address" "0.0.0.0") | indent 10 }}
+
           exec geth \
           --datadir /root/.celo \
           --ipcpath=geth.ipc \
           --nousb \
-          --rpc \
```
Comment (gastonponti) [inline packages/helm-charts/load-test/templates/load-test.statefulset.yaml:116]: `--http` ?

Comment (jcortejoso) [reply packages/helm-charts/load-test/templates/load-test.statefulset.yaml:116]: The `--http` flag is included in the include `common.geth-http-ws-flags`.

**Pull request merged**